app/models/parent_object.rb

```diff
@@ -9,6 +9,8 @@ class ParentObject < ApplicationRecord # rubocop:disable Metrics/ClassLength
   has_many :child_objects, primary_key: 'oid', foreign_key: 'parent_object_oid', dependent: :destroy
   belongs_to :authoritative_metadata_source, class_name: "MetadataSource"
   attr_accessor :metadata_update
+  validates :visibility, inclusion: { in: ['Private', 'Public', 'Yale Community Only'],
+                                      message: "%{value} is not a valid value" }
 
```

Comment:
please add a class method called visibilities like so:

```
def self.visibilities
  ['Private', 'Public', 'Yale Community Only']
end
```
then use that class method instead of the hard coded string here and in the view. That way, if we ever have to add one, we can easily

Reply:
@orangewolf Thank you, that is a good suggestion. I will add a class method.

Reply:
> Looks good on the whole, I think we have an opportunity to reduce some duplication. I'd also like to see a test of the model validation please. I personally think its fine to skip the view spec for this kind of small change, but others may want to weigh in there.

The general DCE pattern is to use system specs for these types of changes, rather than view specs (I don't have a lot of experience with view specs, but they seem a little implementation-focused to me, which I think is a drawback). I do think adding this to the ParentObject system spec would be valuable in this case.